vertx-web/src/main/java/io/vertx/ext/web/package-info.java

```diff
@@ -1794,56 +1794,67 @@
  * redirect should also create a session cookie (or other session mechanism) so the user is not required to authenticate
  * for every request.
  *
- * Due to the nature of OAuth2 spec there are slight changes required in order to use other OAuth2 providers, for
- * example, if you are planning to use Google Auth you implement it as:
+ * Due to the nature of OAuth2 spec there are slight changes required in order to use other OAuth2 providers but
```

Comment:
Good to mention in the docs, clearly.

Perhaps a pointer to some serious article / blog on that topic would be useful as well, but that is really just a minor suggestion. The text as it is now is great.